README.md

```diff
@@ -321,6 +321,8 @@ The following options based on the options for [PouchDB batch fetch](http://pouc
 * `limit`: Maximum number of documents to return.
 * `skip`: Number of docs to skip before returning (warning: poor performance on IndexedDB/LevelDB!).
 
+Also it is possible to add an async option `{async: true|false}` in order to force to sideload or not dependent objects. Please refer to the [Async relationships](#async-relationships) chapter for more details.
```

Comment:
^ It's not clear to me if this means that is overrides in both cases. I.e. if the default is `true` and you set `false` here, it overrides, is that right? And if the default is `false` but you set `true` here, then it also overrides? If that's the case, then the code should probably need to check for `typeof relationOptions.async !== 'boolean'` as well as simple truthy/falsy, since it will be falsy in the case where it's not defined at all.